data/campaigns/Descent_Into_Darkness/scenarios/08_Alone_at_Last.cfg

```
@@ -1080,10 +1083,53 @@
 
         {MOVE_UNIT (id=Malin Keshar) 1 8}
 
-        [message]
-            speaker=Malin Keshar
-            message= _ "The terrain is hazardous in these mountains. Neither the paladins nor Dela will be able to pursue me."
-        [/message]
+        # GitHub Issue #7005: Handle presence/absence of the factions/leaders
+        [if]
+            # Phase one: Before Cadaeus or Dela have arrived
+            # Unlikely, but handle the case for completeness.
+            [variable]
+                name=cadaeus_arrived
+                boolean_equals=no
+            [/variable]
+            [and]
+                [variable]
+                    name=dela_arrived
+                    boolean_equals=no
+                [/variable]
+            [/and]
+            [then]
+                [message]
+                    speaker=Malin Keshar
+                    message= _ "The terrain is hazardous in these mountains. No one should be able to pursue me."
```

Can we just change the line to this for all cases?

```suggestion
                    message= _ "The terrain is hazardous in these mountains. No one will be able to pursue me."
```

---

So just get rid of the references to paladins and Dela altogether? Okay...

---

I think it might be easier this way? It sounds more like text he'd actually say than something more verbose anyway.